api/src/main/java/org/hisp/dhis/dxf2/importsummary/package-info.java

```diff
@@ -3,4 +3,4 @@
 
 import javax.xml.bind.annotation.XmlNs;
 import javax.xml.bind.annotation.XmlNsForm;
-import javax.xml.bind.annotation.XmlSchema;
\ No newline at end of file
+import javax.xml.bind.annotation.XmlSchema;
```

Comment:
Isn't this is the same line changed?

Reply:
There wasn't a newline at the end of the file. I added it.